components/brave_wallet/browser/solana_tx_manager_unittest.cc

```diff
@@ -50,12 +50,30 @@ class SolanaTxManagerUnitTest : public testing::Test {
     keyring_service_.reset(new KeyringService(&prefs_));
     tx_service_.reset(new TxService(json_rpc_service_.get(),
                                     keyring_service_.get(), &prefs_));
+    CreateWallet();
+    AddAccount();
+  }
+
+  void CreateWallet() {
+    base::RunLoop run_loop;
+    keyring_service_->CreateWallet(
+        "brave",
+        base::BindLambdaForTesting([&run_loop](const std::string& mnemonic) {
+          ASSERT_FALSE(mnemonic.empty());
+          run_loop.Quit();
+        }));
+    run_loop.Run();
+  }
 
-    keyring_service_->CreateWallet("testing123", base::DoNothing());
-    base::RunLoop().RunUntilIdle();
-    keyring_service_->AddAccount("Account 1", mojom::CoinType::SOL,
-                                 base::DoNothing());
-    base::RunLoop().RunUntilIdle();
+  void AddAccount() {
+    base::RunLoop run_loop;
+    keyring_service_->AddAccount(
+        "New Account", mojom::CoinType::ETH,
```

Comment:
Shouldn't this be `mojom::CoinType::SOL`?

Reply:
ugh right sorry, fixed